ganga/GangaLHCb/__init__.py

```diff
@@ -54,6 +54,7 @@
                           splitInputDataBySize and splitInputData')
     defaultLHCbDirac = 'prod'
     configLHCb.addOption('LHCbDiracVersion', defaultLHCbDirac, 'set LHCbDirac version')
+    configLHCb.addOption('defaultPlatform', 'x86_64-centos7-gcc8-opt', 'The default platform for applications to use')
```

Comment:
This value is bound to be outdated pretty soon. Is there some way we can set it to be the latest value?

Reply:
This now takes the CMTCONFIG from the standard login by default.